.github/workflows/java.yml

```diff
@@ -41,6 +41,7 @@ jobs:
     - env:
         SPARK_VERSION: ${{ matrix.spark }}
         SCALA_VERSION: ${{ matrix.scala }}
+        SPARK_LOCAL_IP: "127.0.0.1"
```

Comment:
You don't need this line.

Reply:
This is the same workaround of `"spark.driver.bindAddress", "127.0.0.1"` but we only need to set it once, as an env variable.

As you can see [Here](https://github.com/apache/incubator-sedona/pull/516/checks?check_run_id=2344894037) it works.

I only need to apply that to python and the examples CI's.

Also, they have found the problem, and they will work on that on Monday (as described [here](https://github.com/actions/virtual-environments/issues/3185#issuecomment-821758343))